uit/uit.py

```diff
@@ -96,15 +93,8 @@ def __init__(self, ca_file=None, config_file=None, client_id=None, client_secret
         # Environmental variable cache
         self.env = HpcEnv(self)
 
-        if self.config_file is None:
-            self.config_file = DEFAULT_CONFIG_FILE
-        try:
-            with open(self.config_file, 'r') as f:
-                self._config = yaml.safe_load(f)
-        except IOError:
-            pass  # This config file is rarely used, so ignore errors if it doesn't exist
-        except yaml.YAMLError as e:
-            logger.error(f"Error while parsing config file '{self.config_file}': {e}")
+        if config_file:
+            self._config = parse_config(config_file)
```

Comment:
pyuit no longer loads my .uit with the debug_stacktrace_allowlist setting. This is my attempt to load it back (along with an import change above):
```suggestion
        if config_file:
            self._config = parse_config(config_file)
        else:
            self._config = DEFAULT_CONFIG
```

Reply:
I missed that usage. Thanks for catching.